ledger-core/insolar/payload/marker_auth.go

```diff
@@ -38,7 +40,15 @@ func (m *VStateRequest) customSubject() (Reference, authSubjectMode) {
 }
 
 func (m *VCallResult) customSubject() (Reference, authSubjectMode) {
-	return m.GetCallee(), hasAuthSubject
+	switch m.CallType {
```

Comment:
Now I proxy Callee from VCallRequest as is. 

I'm not sure about this behaviour, stop me if I'm wrong
@ivanshibitov 
@cyraxred

Reply:
removed